app/scripts/migrations/052.js

```diff
@@ -0,0 +1,28 @@
+import { cloneDeep } from 'lodash';
+
+import { removeStorageItem } from '../../../ui/lib/storage-helpers';
+
+const version = 52;
+
+/**
+ * Removes storage key no longer required due to move to
+ * fetchWithCache for gas retrieval and storage
+ */
+export default {
+  version,
+  async migrate(originalVersionedData) {
+    const versionedData = cloneDeep(originalVersionedData);
+    versionedData.meta.version = version;
+
+    try {
+      await Promise.all([
```

Comment:
I tried testing this locally, and it seems as though this call is _stalling_, not failing. It hangs forever. It happens with the benchmark and the metric tests because it only affects the first pageload. When you refresh, it's fine.

I found some `localForage` issues that talk about it hanging when trying to open an existing IndexedDB database in  a web worker... maybe something similar is happening here? This is the first time we've used this API in the background. It's only been used in the front-end up to now. :thinking:

Reply:
One solution would be to not `await` this call. Just keep going. That would get past this problem, but it seems a bit... sketchy :grimacing: . Is it possible it would mess with future IndexedDB calls? Who knows.

Maybe we can remove the migration altogether? Having junk data leftover is less bad than freezing the page load. Not ideal, but less bad.

Reply:
Or we could just remove these from local storage, and add a comment to remind us to find a way to remove it from local storage later at some point. Maybe we'll need front-end migrations :thinking:

Reply:
Made an update to see if a value check before attempting to remove will help.

Reply:
It was freezing when _opening_ the database, so I'd be surprised if that helped.